docs/reference/api.yml

```diff
@@ -0,0 +1,3204 @@
+                          type: string
+                    small:
+                      type: object
+                      properties:
+                        en:
+                          type: string
+                    thumb:
+                      type: object
+                      properties:
+                        en:
+                          type: string
+                nutrition:
+                  type: object
+                  properties:
+                    display:
+                      type: object
+                      properties:
+                        en:
+                          type: string
+                        fr:
+                          type: string
+                    small:
+                      type: object
+                      properties:
+                        en:
+                          type: string
+                        fr:
+                          type: string
+                    thumb:
+                      type: object
+                      properties:
+                        en:
+                          type: string
+                        fr:
+                          type: string
+            serving_quantity:
+              type: string
+            serving_size:
+              type: string
+            sortkey:
+              type: integer
+            states:
+              type: string
+            states_hierarchy:
+              type: array
+              items:
+                type: string
+            states_tags:
+              type: array
+              items:
+                type: string
+            stores:
+              type: string
+            stores_tags:
+              type: array
+              items:
+                type: object
+            teams:
+              type: string
+            teams_tags:
+              type: array
+              items:
+                type: string
+            traces:
+              type: string
+            traces_from_ingredients:
+              type: string
+            traces_from_user:
+              type: string
+            traces_hierarchy:
+              type: array
+              items:
+                type: object
+            traces_lc:
+              type: string
+            traces_tags:
+              type: array
+              items:
+                type: object
+            unique_scans_n:
+              type: integer
+            unknown_ingredients_n:
+              type: integer
+            unknown_nutrients_tags:
+              type: array
+              items:
+                type: object
+            update_key:
+              type: string
+            vitamins_prev_tags:
+              type: array
+              items:
+                type: object
+            vitamins_tags:
+              type: array
+              items:
+                type: object
+        status:
+          type: integer
+        status_verbose:
+          type: string
+      x-examples:
+        example-1:
```

Comment:
We really have to split thing (see https://davidgarcia.dev/posts/how-to-split-open-api-spec-into-multiple-files/) because it wont be manageable to have such big files (impossible to review, because it's very hard to know where you are in the file). If we give meaningful titles to files it will help a lot. Eg. this could be "Product_example_1.yml"

Reply:
Also in the future we might want to auto-generate this.